builder/bash/build-slugbuilder-slugrunner.bash

```diff
@@ -0,0 +1,41 @@
+set -eo pipefail
+
+# set debug based on envvar
+[[ $DEBUG ]] && set -x
+
+main() {
+  DOCKER="/usr/bin/docker"
+  DRIVER_OVERRIDE="$(cat /etc/docker.env)"
```

Comment:
@bacongobbler that file is only to replicate this https://github.com/deis/deis/blob/master/builder/image/bin/boot#L56